packages/sitecore-jss-react/src/components/FieldMetadata.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ComponentType, forwardRef } from 'react';
+
+/** The field metadata */
+export interface FieldMetadata {
+  contextItem?: FieldMetadataContextItem;
```

Comment:
Same question as I had during previous review. I think all the field metadata related properties can't be optional, they are required to be present to support chromes hydration. Same note for _FieldMetadataContextItem_